Makefile

```diff
@@ -31,13 +31,17 @@ cflags.dev := -g -O0 -Wall -Wno-trigraphs -Wextra -Wshadow -Wstrict-prototypes -
 cflags.vlg := -g -O0 -Wall -Wno-trigraphs -Wextra
 cflags.cov := -g -O0 -Wall -Wno-trigraphs -Wextra --coverage
 cflags.release := -O3 -flto -funroll-loops -march=native -Wall -Wno-trigraphs
+cflags.dll_dev = -g -O0 -fpic -Wall
+cflags.dll_release = -O3 -fpic -flto -funroll-loops -march=native -Wall -Wno-trigraphs
 
 lflags.cov := --coverage
 
 ldflags.dev := -Llib -pthread $(FSAN_ARG)
 ldflags.vlg := -Llib -pthread
 ldflags.release := -Llib -pthread
-ldflags.cov := -Llib -pthread 
+ldflags.cov := -Llib -pthread
+ldflags.dll_dev := -Llib -pthread $(FSAN_ARG)
```

Comment:
i had a problem using `$(FSAN_ARG)` in a dll, it built fine but when i linked against it and ran the resulting executable i got

```
==396865==ASan runtime does not come first in initial library list; you should either link runtime to your application or manually preload it with LD_PRELOAD.
```

don't know enough about c/asan to diagnose the issue but it's probably enough to just leave the args out for the time being

Reply:
Ah, I have also never seen that issue before, so I agree that it's fine to leave out the FSAN args for now. I removed them in the latest commit.